doc/DeveloperGuide.md

```diff
@@ -88,6 +88,39 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Change name tag
+
+**MSS**
+
+1. User request to change name tag
+2. AddressBook request user to input the name tag user likes to change
+3. User input name tag that he/she likes to change
+4. AddressBook request user's choice of new name tag
+5. User input new name tag
+6. AddressBook request for confirmation
+7. AddressBook update the name tag
```

Comment:
Use cases should always be an interaction. Hence they should look like 1. User 2. AddressBook 3. User 4. Addressbook ... Point 7 cannot be Addressbook as it is not an interaction. Ideally make it part of point 6.